File: selection_sort/selection_sort.c
#include <stdio.h> 

int *selection_sort(int arr[],int len)
{
    int temp;
    int min_idx;
    for(int i = 0;i< len - 1;i++) {
        min_idx = i;
        for (int j = i + 1; j < len; j++)
        {
            if (arr[j] < arr[min_idx])
            {
                min_idx = j;
            }
            
        }
        temp = arr[i];
        arr[i] = arr[min_idx];
        arr[min_idx] = temp;
    }
    return arr;
}

int main (void)
{
    int arr[] = {3,44,38,5,47,15,36,26,27,2,46,4,19,50,48};
    int len = sizeof(arr)/sizeof(int);
    selection_sort(arr,len);
    for (int i = 0; i < len; i++)
    {
        printf("%d\t",arr[i]);
    }

    return 0;
}

File: selection_sort/selection_sort.js
function selection_sort(arr) {
    let min_idx,temp;
    for (let i = 0; i < arr.length-1; i++) {
        min_idx = i;
        for (let j = i + 1; j < arr.length; j++) {
            if (arr[j] < arr[min_idx]) {
                min_idx = j
            }
        }

        temp = arr[i]
        arr[i] = arr[min_idx]
        arr[min_idx] = temp
    }
    return arr
}

const arr = [3,44,38,5,47,15,36,26,27,2,46,4,19,50,48]
console.log(selection_sort(arr))